src/tests/perf-synthetic.test.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console, jest/expect-expect */
+import { join } from "path";
+
+import { Connection, DuckDB } from "@addon";
+
+import { writeSyntheticParquetFile } from "./synthetic-test-data-generator";
+
+const filePath = join(__dirname, "../large-synth");
```

Comment:
Should we exclude this location from .gitignore?

Reply:
it's already inside `dist` which is excluded

Reply:
Ah, okay